main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # 1. Create:
+    genres_for_create = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+    actors_for_create = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre in genres_for_create:
+        Genre.objects.create(
+            name=genre,
+        )
+
+    for first_name, last_name in actors_for_create:
+        Actor.objects.create(
+            first_name=first_name, last_name=last_name,
+        )
```

Comment:
You are using a `for` loop to create Actor objects, which is good as it adheres to point #3 in the checklist. Also, you are using a list of tuples for better iterating, which adheres to point #2 in the checklist.